types.go

```diff
@@ -261,7 +261,12 @@ type VSPTicketPurchaseInfo struct {
 
 /** end ticket-related types */
 
-/** begin politea proposal types */
+/** begin politeia types */
+type Err struct {
```

Comment:
This struct doesn't seem to have a use outside `politeia_client.go`, it can be declared there instead.